src/main/java/org/bonitasoft/connectors/ws/SecureWSConnector.java

```diff
@@ -244,6 +279,58 @@ protected void executeBusinessLogic() throws ConnectorException {
         setOutputParameter(OUTPUT_RESPONSE_DOCUMENT_BODY, responseDocumentBody);
     }
 
+    private void sanitizeInputs() {
+
+        Map<String, Object> sanitizedInputs = new HashMap<>();
+        sanitizedInputs.put(HTTP_HEADERS, sanitizeHTTPHeaders((List<List<Object>>) getInputParameter(HTTP_HEADERS)));
+        sanitizedInputs.put(PASSWORD, sanitizeString((String) getInputParameter(PASSWORD)));
```

Comment:
Why not directly `setInputParameter(PASSWORD, sanitizeString((String) getInputParameter(PASSWORD)));` ?

Reply:
because setInput_Parameter_ does not exist ?